File: python/codes/L4_parameterized_constructor.py
class Book:
    def __init__(self, title, author):
        self.title = title
        self.author = author

    def display(self):
        print(f"Title: {self.title}, Author: {self.author}")

my_book = Book("The Lord of the Rings", "J.R.R. Tolkien")
my_book.display()

File: javascript/codes/L4_parameterized_constructor.js
class Book {
    constructor(title, author) {
        this.title = title;
        this.author = author;
    }

    display() {
        console.log(`Title: ${this.title}, Author: ${this.author}`);
    }
}

const myBook = new Book("The Lord of the Rings", "J.R.R. Tolkien");
myBook.display();
